internal/meta/jsonfiledb/readtx.go

```diff
@@ -15,12 +15,15 @@ func (tx *readTx) Repository() (meta.Repository, bool) {
 	return tx.state.RepositoryState, tx.state.RepositoryState.ID != ""
 }
 
-func (tx *readTx) Branch(name string) (branch meta.Branch, ok bool) {
-	branch, ok = tx.state.BranchState[name]
-	if !ok {
+func (tx *readTx) Branch(name string) (meta.Branch, bool) {
```

Comment:
Also unrelated, was trying to figure out bug above, but thought it fine to leave this in